src/parsers/index.js

```diff
@@ -0,0 +1,20 @@
+import cucumberParser from './cucumber'
+import multiTestParser from './multitest'
+import standardParser from './standard'
+
+let all = [cucumberParser, multiTestParser, standardParser]
+
+function getParser (name, output = '') {
+  let parser
+  if (name) {
+    parser = all.find((parser) => name === parser.name)
+    if (!parser) { throw new Error(`Invalid Parser Specified: ${name}`) }
+  } else {
+    parser = all.find((p) => {
+      return p.test(output)
+    })
```

Comment:
Got some doubts about this last part (trying to discover the correct parser when none was specified). It is very unclear to users how this is done (magic). I know that protractor-flake did this before, but now that we have a new parser (total 3) I propose a cleaner solution: we select standardParser by default and let the user decide which parser to use. This will make the break in the change a worse (for cucumber users), but I do think this is the cleanest option.

If you still want to keep this automatic detection in protractor-flake, I propose having the test function of multitestParser return false by default. This might even turn this change in non-breaking, because the multitestParser will never be used by default.

So in short I see 2 paths:
1) make the break worse, but reduce the magic done by selecting standardParser by default.
2) fix the break by never selecting multitestParser by default (have it's test function return false by default).